kubelet/tests/test_kubelet.py

```diff
@@ -121,6 +121,10 @@
     'kubernetes.kubelet.container.log_filesystem.used_bytes',
     'kubernetes.kubelet.pod.worker.duration.sum',
     'kubernetes.kubelet.pod.worker.duration.count',
+    'kubernetes.kubelet.pleg.relist_duration.count',
+    'kubernetes.kubelet.pleg.relist_duration.sum',
+    'kubernetes.kubelet.pleg.relist_interval.count',
+    'kubernetes.kubelet.pleg.relist_interval.sum',
```

Comment:
could we test `kubelet.pleg.discard_events` as well?

Reply:
Tests for those metrics added : https://github.com/DataDog/integrations-core/pull/11616/commits/e26d8594e0a0d7d5346c0ef66bba0cdc8223d5d3#diff-6e91032eac26fac940963e886e2e59b954453ddeed1cd25a991310b0328b215fR153-R154